### DISCUSSION TITLE
Shadowcopy

### DISCUSSION THREAD

**Discussion by sevenam** [thomhurst/TUnit#696] [General]
How can you turn off shadowcopy in TUnit?

In xunit it can be done with a json file called `xunit.runner.json` like this:

{
  "shadowCopy": false
}

**Comment by thomhurst** [comment]
@sevenam can you give more details around your scenario and what problems/limitations you're facing?

I haven't built anything regarding that currently.

**Reply by sevenam** [reply]
Was trying out TUnit with an existing project that has been a bit problematic in the past. The project had shadowcopy set to false in xunit. Was unfortunately no good commit message on the commit where it was added, so I am unsure of the reason behind it. It does however seem to work fine with TUnit, so possibly no issue!

**Reply by thomhurst** [reply]
Happy to close this if you've got no issues?

**Reply by sevenam** [reply]
Sure!
